src/pages/orders/components/OrderDetailsCard/OrderDetailsCardFooter/OrderDetailsCardFooter.tsx

```diff
@@ -103,16 +102,18 @@ const OrderDetailsCardFooter = ({ sendFile }: { sendFile: (file: File) => void }
     }, [verificationCode]);
 
     useEffect(() => {
-        handleModalDisplay(error?.error?.code);
+        // @ts-expect-error types are not correct from api-hooks
+        handleModalDisplay(error?.code);
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [
-        error?.error,
+        // @ts-expect-error types are not correct from api-hooks
+        error?.code,
         isBuyOrderForUser,
         isError,
         isSuccess,
-        verificationNextRequest,
+        verificationTokenExpiry,
         data?.is_dry_run_successful,
-        orderDetails?.status,
+        orderDetails.status,
```

Comment:
wont removing the optional chaining cause runtime error

Reply:
ah yeah i can just add it back, i forgot to undo this change here